packages/merge-styles/src/Stylesheet.ts

```diff
@@ -91,30 +76,34 @@ export interface IStyleSheetConfig {
    * Initial value for classnames cache. Key is serialized css rules associated with a classname.
    */
   classNameCache?: { [key: string]: string };
+
+  ownerWindow?: Window;
+
+  inShadow?: boolean;
+
+  currentStylesheetKey?: string;
 }
 
 /**
  * Representation of Stylesheet used for rehydration.
  */
 export interface ISerializedStylesheet {
   classNameToArgs: Stylesheet['_classNameToArgs'];
-  counter: Stylesheet['_counter'];
+  counter: Stylesheet['_styleCounter'];
   keyToClassName: Stylesheet['_keyToClassName'];
   preservedRules: Stylesheet['_preservedRules'];
   rules: Stylesheet['_rules'];
 }
 
 const STYLESHEET_SETTING = '__stylesheet__';
 
-const ADOPTED_STYLESHEETS = '__mergeStylesAdoptedStyleSheets__';
-
 /**
  * MSIE 11 doesn't cascade styles based on DOM ordering, but rather on the order that each style node
  * is created. As such, to maintain consistent priority, IE11 should reuse a single style node.
  */
 const REUSE_STYLE_NODE = typeof navigator !== 'undefined' && /rv:11.0/.test(navigator.userAgent);
 
-const SUPPORTS_CONSTRUCTIBLE_STYLESHEETS = typeof window !== 'undefined' && 'CSSStyleSheet' in window;
+// const SUPPORTS_CONSTRUCTIBLE_STYLESHEETS = typeof window !== 'undefined' && 'CSSStyleSheet' in window;
```

Comment:
Leaving this here as I'll need it again when I add polyfilled support for constructable sheets.